packages/components/src/DateTimePickers/pickers/YearPicker/YearPicker.test.js

```diff
@@ -4,11 +4,101 @@ import { shallow } from 'enzyme';
 import YearPicker from './YearPicker.component';
 
 describe('YearPicker', () => {
-	it('should render a YearPicker', () => {
-		// when
-		const wrapper = shallow(<YearPicker />);
+	it('should render', () => {
+		const wrapper = shallow(<YearPicker
+			yearSelected={2012}
+		/>);
 
-		// then
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
+
+	it('should have exactly one selected year', () => {
+		const wrapper = shallow(<YearPicker
+			yearSelected={2012}
+		/>);
+
+		const actions = wrapper.findWhere(n => n.name() === 'PickerAction');
+
+		expect(actions.length).toBe(5);
+
+		const unselectedActions = actions.filterWhere(action => action.prop('isSelected') === false);
+		const selectedActions = actions.filterWhere(action => action.prop('isSelected') === true);
+
+		expect(unselectedActions.length).toBe(4);
+		expect(selectedActions.length).toBe(1);
+
+		unselectedActions.forEach(action => {
```

Comment:
Please apply the same remarks as for Months tests

Reply:
✔